app/2.0/docs/tools/documentation.md

```diff
@@ -4,10 +4,302 @@ title: Document your elements
 
 <!-- toc -->
 
-Polymer 2.x code can be documented using JSDoc-style comments, like Polymer 1.x code.
-Documentation tools for Polymer 2.x are still under development. Information about the
-new documentation tools will be available in a future release.
+You can provide API docs for Polymer custom elements by writing
+documentation comments in your source files. Using the `iron-component-pages` element,
+you can create a simple documentation page for your elements that parses these comments
```

Comment:
Nit: Might want to reword this slightly, because`iron-component-page` doesn't itself parse the comments (the 1.0 version used to embed Hydrolysis, so this used to be true).